.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "airbnb",
   "rules": {
-    "linebreak-style": "off"
+    "linebreak-style": "off",
+    "no-underscore-dangle": "off"
```

Comment:
If you're going to make changes to things outside of the scope of the ticket, could you provide some context to them? For example, what is `no-underscore-dangle` and why did you turn this off?

Reply:
Sorry about that. I should have commented. See this comment:  https://github.com/donthedeveloper/site-builder/pull/84/files/0773b389c76bacda3dff5143a150231e4eafd697#r262762370

Let me know if you think that makes sense, if you don't mind.